server/errorstore/errors_test.go

```diff
@@ -118,20 +113,18 @@ func TestHashErrEris(t *testing.T) {
 	})
 
 	t.Run("HashNew", func(t *testing.T) {
-		err := alwaysErisErrors()
-		werr := eris.Wrap(err, "wrap eris")
-		werr1, werr2 := pkgErrors.Wrap(err, "wrap pkg"), fmt.Errorf("wrap fmt: %w", err)
+		err := alwaysFleetErrors()
+		werr0, werr1 := pkgErrors.Wrap(err, "wrap pkg"), fmt.Errorf("wrap fmt: %w", err)
 		wantHash := hashError(err)
-		h0, h1, h2 := hashError(werr), hashError(werr1), hashError(werr2)
```

Comment:
similar to the comment above, I would not expect `werr` here and `err` to hash to the same value, for example I would expect the error returned by the function `a` below to be hashed differently than the error returned by `b`

```go
func ListSoftware() error {
  // ...
  return errors.New("stdlib error")
}

func a() error {
  if err := ListSoftware() {
    return ctxerr.Wrap(ctx, err, "from a")
  }


func b() error {
  if err := ListSoftware() {
    return ctxerr.Wrap(ctx, err, "from b")
  }
}
```

Reply:
I agree - that's how it should be, the stack trace of the root error in that case is different in the two scenarios, one is a->ListSoftware, the other is b->ListSoftware.